### ISSUE TITLE
Logfiles missing error messages

### ISSUE THREAD

**Issue by despens** [ONLYOFFICE/DocumentServer#328]
**Do you want to request a *feature* or report a *bug*?**

Not clear if feature request or missing documentation

**What is the current behavior?**

As deescribed in [another issue](https://github.com/ONLYOFFICE/onlyoffice-owncloud/issues/191), integration with nextcloud yields the error message `Error occurred in the document service: Error while downloading the document file to be converted.` I am trying to debug the problem, but the DocumentServer doesn't seem to record an error log with this information.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

- Installed DocumentServer in an LXC container on Ubuntu 18.04 via the debian repositories, [following instruactions](https://helpcenter.onlyoffice.com/server/linux/document/linux-installation.aspx).
- Installing the nextcloud ONLYOFFICE integration pluging from the nextcloud appstore.
- Use caddy server to provide reverse HTTPS proxy for the DocumentServer and nextcloud; both operate when accessing them from their public URLs.
- When setting up the integration plugin in nextcloud, the error message `Error occurred in the document service: Error while downloading the document file to be converted.` appears.
- Inside the ONLYOFFICE lxc container, log files available in `/var/log` do not contain an error message or exmplanation.

**What is the expected behavior?**
A log file should contain the error message and further debugging information so I can track down the error.

**Did this work in previous versions of DocumentServer?**
I did not use earlier versions.

**DocumentServer version:** 5.1.4-22

**Operating System:** Server and client: Ubuntu 18.04

**Browser version:**
Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0

**Comment by ShockwaveNN** [comment]
@despens could you attach whole /var/log/onlyoffice dir to make sure everything seems to be fine in logs?

**Comment by despens** [comment]
Thank you @ShockwaveNN! I wouldn't want to post the whole log directory publicly since I am not sure if it contains private information. How can I get the tar.gz file to you?

**Comment by ShockwaveNN** [comment]
@despens send it to shockwavenn@gmail.com

**Comment by despens** [comment]
Pavel, thank you so much for offering to look at my log files. Before sending the tarball I took a last look and found the issue in `/var/log/onlyoffice/documentserver/converter/out.log`: nextcloud didn't accept the internal connection URL. I had to add to nextcloud's configuration.

The nextcloud integration app I had to set up as follows:

Document Enditing Service: `https://office.public.nextcloud.address`
Advanced server settings:
Document Editing Service address for internal requests from the server: `http://onlyoffice.lxc`
Server address for internal requests from the Document Editing Service: `http://nextcloud.lxd/`
Secret key: `verysecret`

In the nextcloud `config.php` file I had to add the internal container name to trusted domains:

```php
  'trusted_domains' => 
  array (
    0 => 'public.nextcloud.address', // publicly accessible domains
    1 => 'nextcloud.lxd' // internal container URL
  )
```

Now everything works wonderfully!

I would suggest to add the information about trusted domain setup to the ONLYOFFICE installation instruction manual.
